### PR TITLE
doc: manual override of SML tables

### PR DESCRIPTION
**Pull request by b-gent**
Some further manual updates as a result
of broken script.

### CODE REVIEW

**Review by greg-fer — APPROVED** [review]

**Review by divipillai — APPROVED** [review]

File: doc/nrf/releases_and_maturity/software_maturity.rst [hunk 1]
```diff
@@ -171,7 +171,7 @@ Protocol support
 The following table indicates the software maturity levels of the support for each :ref:`protocol <protocols>`:
 
 .. sml-table:: top_level
-   :insert-values: [("Bluetooth","nRF54H20","Experimental"), ("Matter","nRF54H20","Experimental"), ("Thread","nRF54H20","Experimental"), ("Wi-Fi","nRF54H20","Experimental"),("Bluetooth","nRF54L15","Experimental"), ("Bluetooth Mesh","nRF54L15","Experimental"), ("Matter","nRF54L15","Experimental"), ("Sidewalk","nRF54L15","Experimental"), ("Thread","nRF54L15","Experimental"), ("Wi-Fi","nRF54L15","Experimental")]
+   :insert-values: [("Bluetooth","nRF54H20","Experimental"), ("Matter","nRF54H20","Experimental"), ("Thread","nRF54H20","Experimental"), ("Wi-Fi","nRF54H20","Experimental"),("Bluetooth","nRF54L15","Experimental"), ("Bluetooth Mesh","nRF54L15","Experimental"), ("Matter","nRF54L15","Experimental"), ("Sidewalk","nRF54L15","Experimental"), ("Thread","nRF54L15","Experimental"), ("Wi-Fi","nRF54L15","Experimental"), ("DECT NR+ PHY","nRF9160","-"), ("DECT NR+ PHY","nRF9161","Experimental"), ("DECT NR+ PHY","nRF9151","Experimental")]
```
Comment (divipillai) [inline doc/nrf/releases_and_maturity/software_maturity.rst:174]:
```suggestion
   :insert-values: [("Bluetooth","nRF54H20","Experimental"), ("Matter","nRF54H20","Experimental"), ("Thread","nRF54H20","Experimental"), ("Wi-Fi","nRF54H20","Experimental"),("Bluetooth","nRF54L15","Experimental"), ("Bluetooth Mesh","nRF54L15","Experimental"), ("Matter","nRF54L15","Experimental"), ("Sidewalk","nRF54L15","Experimental"), ("Thread","nRF54L15","Experimental"), ("Wi-Fi","nRF54L15","Experimental"), ("DECT_PHY","nRF9160","-"), ("DECT_PHY","nRF9161","Experimental"), ("DECT_PHY","nRF9151","Experimental")]
```
Is there any way to rename DECT_PHY to DECT NR+ PHY in the table?

**Review by divipillai — CHANGES_REQUESTED** [review]

**Comment by b-gent** [comment]
Looks like the :insert-values: method does not really meet our requirements. I will be changing these tables to manual markdown tables in the next commit.

File: doc/nrf/releases_and_maturity/software_maturity.rst [hunk 2]
```diff
@@ -231,7 +250,25 @@ The following table indicates the software maturity levels of the support for ea
...
+   | Feature                                       | nRF52810 | nRF52811 | nRF52820 | nRF52832 | nRF52833 | nRF52840    | nRF5340          | nRF54H20       | nRF54L15       | nRF9151 | nRF9160        | nRF9161        |
+   +===============================================+==========+==========+==========+==========+==========+=============+==================+================+================+=========+================+================+
+   | **Bluetooth LE Coexistence**                 | -        | -        | -        | -        | -        | Experimental| Supported<sup>1</sup>| -            | -            | -       | -              | -              |
+   +-----------------------------------------------+----------+----------+----------+----------+----------+-------------+------------------+----------------+----------------+---------+----------------+----------------+
+   | **Thread Coexistence**                        | -        | -        | -        | -        | -        | -           | Experimental     | -              | -              | -       | -              | -              |
```
Comment (VivekUppunda) [inline doc/nrf/releases_and_maturity/software_maturity.rst:258]: This is not visible in the Documentation Link generated for this PR. Will this be seen in the documentation for main build?

File: doc/nrf/releases_and_maturity/software_maturity.rst [hunk 3]
```diff
@@ -231,7 +250,25 @@ The following table indicates the software maturity levels of the support for ea
...
+   | **STA Mode**                                  | -        | -        | -        | -        | -        | Experimental<sup>3</sup>| Supported<sup>1</sup>| Experimental<sup>4</sup>| Experimental<sup>5</sup>| -       | -              | -              |
+   +-----------------------------------------------+----------+----------+----------+----------+----------+-------------+------------------+----------------+----------------+---------+----------------+----------------+
+   | **Scan only (for location accuracy)**         | -        | -        | -        | -        | -        | Experimental<sup>6</sup>| Supported<sup>7</sup>| -            | -            | Supported| Supported<sup>6</sup>| Supported<sup>6</sup>|
+   +-----------------------------------------------+----------+----------+----------+----------+----------+-------------+------------------+----------------+----------------+---------+----------------+----------------+
+   | **SoftAP Mode (for Wi-Fi provisioning)**      | -        | -        | -        | -        | -        | -           | Supported        | -              | -              | -       | -              | -              |
```
Comment (VivekUppunda) [inline doc/nrf/releases_and_maturity/software_maturity.rst:268]: This is change is also not visible in the PR documentation link

**Review by VivekUppunda — CHANGES_REQUESTED** [review]

**Review by VivekUppunda — APPROVED** [review]

**Review by divipillai — APPROVED** [review]

**Pull request merged**